pkg/proof/README.md

```diff
@@ -0,0 +1,201 @@
+# Proof
+
+This package enables proof queries for Celestia transactions and blobs.
+
+## Details
+
+The following section will cover transaction inclusion proofs and blob inclusion proofs.
```

Comment:
[nit] this section doesn't add any value. After reading it, I would expect two headings below it called:

```markdown
### Transaction inclusion proof

// content

### Blob inclusion proof

// content
```

but since that's not the case, this whole section could be deleted.

```suggestion
```